### PR TITLE
Add webinarium/symfony-lazysec package

### PR DESCRIPTION
**Pull request by webinarium**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: webinarium/symfony-lazysec/2.0/config/packages/lazysec.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+services:
+    _defaults:
+        autowire:      true
```
Comment (Nyholm) [inline webinarium/symfony-lazysec/2.0/config/packages/lazysec.yaml]: Do you really need _defaults and autoworker when you only congure one service?

(I have not looked at the bundle, maybe you do)

**Comment by webinarium** [comment]
> Do you really need _defaults and autoworker when you only congure one service?

Hi @Nyholm,

Thank you for your feedback. You are absolutely right - no need for the defaults, so I simplified the config file there.

Best regards,
Artem

File: webinarium/symfony-lazysec/2.0/config/packages/lazysec.yaml (hunk 1 above)
Comment (webinarium) [reply webinarium/symfony-lazysec/2.0/config/packages/lazysec.yaml]: Thank you for your feedback. You are absolutely right - no need for the defaults, so I have simplified the config file and made another commit.

**Comment by nicolas-grekas** [comment]
shouldn't you also add a commented line like this one, with a comment?
`#    LazySec\Repository\UserRepositoryInterface: '@App\Repository\UserRepository'`

**Comment by webinarium** [comment]
@nicolas-grekas, so true!
I didn't add that line because it would break an app without a repository prepared first, but it's slipped from my mind that I can add this line commented. :) Thank you for the hint, I have updated the config and pushed the update.

**Comment by webinarium** [comment]
If there is anything else needed to get this PR merged, please let me know.

**Pull request merged**